lib/srv/desktop/rdp/rdpclient/src/rdpdr/consts.rs

```diff
@@ -14,6 +14,18 @@
 
 pub const CHANNEL_NAME: &str = "rdpdr";
 
+// Each redirected device requires a unique ID.
+pub const SCARD_DEVICE_ID: u32 = 1;
```

Comment:
> What do we use for a unique ID for the redirected folders?

Currently I'm using `2`. This is generated and sent by the TDP Client in the `SharedDirectoryAnnounce` message. It will fail if we send `SCARD_DEVICE_ID` because a `SharedDirectoryAnnounce` ultimately results in

https://github.com/gravitational/teleport/blob/332af8563418474fa93401b12746fbf5cbd9c552/lib/srv/desktop/rdp/rdpclient/src/rdpdr/mod.rs#L263-L279

being called, which calls

https://github.com/gravitational/teleport/blob/332af8563418474fa93401b12746fbf5cbd9c552/lib/srv/desktop/rdp/rdpclient/src/rdpdr/mod.rs#L294-L303

which disallows duplicates.

I'm basically banking on myself remembering that we can send anything other than `1` when implementing the TDP Client, as an idiosyncrasy of our implementation.

> Additionally, should we keep the context from the original comment that we hardcode this to 1 because we only ever emulate a single smart card device?

```
// Each redirected device requires a unique ID. We only share
// one permanent smartcard device, so we can give it hardcoded ID 1.
```